policy-service/src/component-tests/java/uk/gov/gchq/palisade/component/policy/service/RedisPolicyCachingTest.java

```diff
@@ -1,142 +0,0 @@
-/*
- * Copyright 2020 Crown Copyright
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package uk.gov.gchq.palisade.component.policy.service;
-
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
-import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.ContextConfiguration;
-
-import uk.gov.gchq.palisade.contract.policy.PolicyTestData;
-import uk.gov.gchq.palisade.policy.IsTextResourceRule;
-import uk.gov.gchq.palisade.resource.Resource;
-import uk.gov.gchq.palisade.resource.impl.FileResource;
-import uk.gov.gchq.palisade.resource.impl.SystemResource;
-import uk.gov.gchq.palisade.service.policy.PolicyApplication;
-import uk.gov.gchq.palisade.service.policy.service.PolicyService;
-import uk.gov.gchq.palisade.service.policy.service.PolicyServiceCachingProxy;
-import uk.gov.gchq.palisade.service.request.Policy;
-
-import java.util.Optional;
-import java.util.concurrent.TimeUnit;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-@ActiveProfiles("redis")
-@SpringBootTest(
-        classes = {PolicyApplication.class},
-        webEnvironment = WebEnvironment.NONE,
-        properties = {"spring.cache.redis.timeToLive=1s"}
-)
-@ContextConfiguration(classes = {RedisTestConfiguration.class})
-class RedisPolicyCachingTest extends PolicyTestData {
```

Comment:
What happened to this test? Why is it gone?

Reply:
its moved to RedisPersistenceContractTest to keep up with naming conventions on the other services redis tests, git is just being lazy and not showing the re-write